services/nginz/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Requires docker >= 17.05 (requires support for multi-stage builds)
-FROM alpine:3.10 as libzauth-builder
+FROM alpine:3.11 as libzauth-builder
```

Comment:
Nevermind the change got reverted here again. I was reading this from commit to commit. Ignore previous comment :)